WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -83,9 +84,17 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         ((WordPress) getActivity().getApplication()).component().inject(this);
 
-        addPreferencesFromResource(R.xml.notifications_settings);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
I'm concerned about the long-term maintenance of having two separate prefs files. Can we stick with a single prefs file and simply hide the sights & sounds category?

Reply:
Thanks for the clarification @nbradbury and @theck13 ! Didn't know you could do that, and my exploration showed not to be as thorough in this case. Will implement accordingly as suggested on our chat and comment back here 🙇